db/migrations/0003_order_user.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.0.2 on 2024-12-24 16:38
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_order_alter_movie_actors_alter_movie_genres_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='order',
+            name='user',
+            field=models.ForeignKey(default=1, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
```

Comment:
The default value for the `user` field is set to `1`. Ensure that a user with ID `1` exists in your database, or consider using a different approach to set this field for existing records.